client/me/help/controller.js

```diff
@@ -42,7 +42,7 @@ export function help( context, next ) {
 	// FIXME: Auto-converted from the Flux setTitle action. Please use <DocumentHead> instead.
 	context.store.dispatch( setTitle( i18n.translate( 'Help', { textOnly: true } ) ) );
 
-	context.primary = <HelpComponent isCoursesEnabled={ config.isEnabled( 'help/courses' ) } />;
+	context.primary = <HelpComponent />;
```

Comment:
Note: The only reference to `isCoursesEnabled` was removed last month in 75a868f033fb1, so the prop not being passed anymore should be fine.